packages/common/src/store/purchase-content/sagas.ts

```diff
@@ -234,62 +292,29 @@ function* doStartPurchaseContentFlow({
     const extraAmountBN = new BN(extraAmount ?? 0).mul(BN_USDC_CENT_WEI)
     const totalAmountDueCentsBN = priceBN.add(extraAmountBN) as BNUSDC
```

Comment:
I started attempting to update some of this to `FixedDecimal` and it immediately became scary. So I chose to leave it alone for now until we can convert all of the code using formatting and calculation utilities (such as the `ceilingBNUSDCToNearestCent()` call below)

Reply:
I think keeping these as `BNUSDC` or `UsdcWei` makes sense while doing some math. I don't think we should have `FixedDecimal`'s being math'ed on or passed around.

Worth noting though that every wallet.ts/formatUtils.ts file dealing with currency has an equivalency using `FixedDecimal` - you can find it in their respective tests:
https://github.com/AudiusProject/audius-protocol/blob/main/packages/common/src/utils/wallet.test.ts
https://github.com/AudiusProject/audius-protocol/blob/main/packages/common/src/utils/formatUtil.test.ts